salt/redash/init.sls

```diff
@@ -1,16 +1,24 @@
+{% from 'lib.sls' import create_pg_database %}
+
 include:
   - docker
 
 # =================================================================================================
 # See https://ocdsdeploy.readthedocs.io/en/latest/deploy/redash.html for installation instructions.
 # =================================================================================================
 
```

Comment:
I think this can now be removed? (And the linked docs updated.)

Reply:
Header removed.